src/VirtualTree.lua

```diff
@@ -0,0 +1,86 @@
+local createReconciler = require(script.Parent.createReconciler)
+local RobloxRenderer = require(script.Parent.RobloxRenderer)
+local shallow = require(script.Parent.shallow)
+local Symbol = require(script.Parent.Symbol)
+local Type = require(script.Parent.Type)
+
+local config = require(script.Parent.GlobalConfig).get()
+
+local DEFAULT_RENDERER = createReconciler(RobloxRenderer)
```

Comment:
should the reconciler be passed to the tree instead of having all mount calls have to use the same reconciler?

Reply:
Yes the reconciler is passed through the `options` table. My comment above gives more detail about that